app/src/main/java/net/squanchy/search/view/ItemsAdapter.java

```diff
@@ -0,0 +1,41 @@
+package net.squanchy.search.view;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import net.squanchy.search.model.TitledList;
+import net.squanchy.speaker.domain.view.Speaker;
+
+import static net.squanchy.search.view.SpeakerAdapter.ViewTypeId;
+
+class ItemsAdapter {
+
+    private final List<TitledList<Speaker>> items = new ArrayList<>();
```

Comment:
I suppose this will have to change when we add the categories (probably just two lists is enough), but good for now